substrate/frame/revive/rpc/src/lib.rs

```diff
@@ -252,12 +252,7 @@ impl EthRpcServer for EthRpcServerImpl {
 			.client
 			.dry_run(&transaction, block.unwrap_or_else(|| BlockTag::Latest.into()))
 			.await?;
-		let output = dry_run.result.map_err(|err| {
-			log::debug!(target: LOG_TARGET, "Dry run failed: {err:?}");
-			ClientError::DryRunFailed
-		})?;
-
-		Ok(output.into())
+		Ok(dry_run.result.into())
```

Comment:
I can't find this `From` implementation. Would be interested in how the error is converted.

Reply:
Bytes is just wrapper for `Vec<u8>`

Reply:
https://github.com/paritytech/polkadot-sdk/blob/pg/revive-fixes/substrate/frame/revive/src/evm/api/byte.rs?plain=1#L111-L115